snapcraft/plugins/_python/_python_finder.py

```diff
@@ -0,0 +1,69 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import glob
+
+from . import errors
+
+
+def get_python_command(python_version, stage_dir, install_dir):
+    """Find the python command to use, preferring staged over the part.
+
+    :param str python_version: The python version to find (python2 or python3)
+    :param str stage_dir: Path to the staging area
+    :param str install_dir: Path to the part's install area
+
+    :return: Path to the python command that was found
+    :rtype: str
+
+    :raises MissingPythonCommandError: If no python could be find in the
+                                       staging or part's install area.
+    """
+
```

Comment:
I think we need to agree on this empty line. I find it unnecessary.

Reply:
I find it annoying on one-liner dockstrings, but unnecessary on multi-line. No strong preference, I'll remove them.

Reply:
Done.